portfolio/src/main/webapp/index.html

```diff
@@ -1,47 +1,68 @@
 <!DOCTYPE html>
 <html>
- 
+
 <head>
-    <meta charset="UTF-8">
-    <title>Kyra Acquah's Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <script src="script.js"></script>
+	<meta charset="UTF-8">
+	<title>Kyra Acquah | Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<script src="script.js"></script>
 </head>
- 
+
 <body>
-    <nav class="navbar">
-        <ul class="navbar-nav">
-            <li class="nav-item">
-                <a href="#" class="nav-link">
-                    <i class="large material-icons">portrait</i>
-                    <span class="link-text">About me</span>
-                </a>
-            </li>
-            <li class="nav-item">
-                <a href="#" class="nav-link">
-                    <i class="large material-icons">school</i>
-                    <span class="link-text">Education</span>
-                </a>
-            </li>
-            <li class="nav-item">
-                <a href="#" class="nav-link">
-                    <i class="large material-icons">laptop_chromebook</i>
-                    <span class="link-text">Projects</span>
-                </a>
-            </li>
-            <li class="nav-item">
-                <a href="#" class="nav-link">
-                    <i class="large material-icons">work</i>
-                    <span class="link-text">Work experience</span>
-                </a>
-            </li>
-        </ul>
-    </nav>
-    <main>
-        <h1>Kyra Acquah's Portfolio</h1>
-        <p>Hello, welcome to my portfolio!</p>
-    </main>
+	<nav class="navbar">
```

Comment:
This is a very small thing, but Google's style guide recommends using 2 spaces for indentation, not 4. I think in this case, that would be helpful because there are a lot of levels of indentation, so it ends up looking quite spread out. (However, I can't figure out how to set this as default in cloud shell, so let me know if you do!)

Reply:
Thanks! I was able to change it to 2 by clicking where is says "Spaces" at the bottom right of the editor and changing it from 4 to 2.